File: Lecture-7/AlphabetDiamond.java
public class AlphabetDiamond {
    public static void main(String[] args) {
        int row = 5;
        int s = 'A';
        
        for (int i = 1; i <= row; i++) {

            for (int k = row; k > i; k--) {

                System.out.print(" ");
            }

            for (int j = 0; j < (i * 2) - 1; j++) {
                System.out.print((char) (s+j));
            }

            System.out.println();
        }

        for (int i = row - 1; i > 0; i--) {

            for (int k = row; k > i; k--) {

                System.out.print(" ");
            }

            for (int j = 0; j < (i * 2) - 1; j++) {
                System.out.print((char) (s+j));
            }

            System.out.println();
        }
    }
}

File: DS-Algo-with-Python/Lecture-7/AlphabetDiamond.py
row = 5
s = 'A'

for i in range(1, row + 1):
    for k in range(row, i, -1):
        print(" ", end="")
    for j in range(0, (i * 2)-1):
        print(chr(ord(s) + j), end="")
    print("")


for i in range(row - 1, 0 , -1):
    for k in range(row, i, -1):
        print(" ", end="")

    for j in range((i * 2) - 1):
        print(chr(ord(s) + j), end="")
    
    print("")
